moveit_py/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>moveit_py</name>
+  <version>0.0.0</version>
```

Comment:
Is this package actually going to be versioned? I would think yes.

Maybe more of a @henningkayser question.

Reply:
I left it as a placeholder, I would prefer to have versioning for the library to make it more official and easier for users to reference.